extensions/amp-story/1.0/amp-story-store-service.js

```diff
@@ -286,8 +289,11 @@ const actions = (state, action, data) => {
             [StateProperty.CURRENT_PAGE_ID]: data.id,
             [StateProperty.CURRENT_PAGE_INDEX]: data.index,
           }));
+    case Action.SET_TOTAL_PAGES:
+      return /** @type {!State} */ (Object.assign(
+          {}, state, {[StateProperty.TOTAL_PAGES]: data}));
     default:
-      dev().error(TAG, `Unknown action ${action}.`);
+      dev().error(TAG, 'Unknown action %s.', action);
```

Comment:
Is there any reason why this is better? If not, I think we should stick to the official JavaScript styleguide and keep using string literals.

Reply:
The linter is complaining about this. I think it's a recent addition to it.

/cc @jridgewell

Reply:
Sounds good : )